website/docs/Pipelines/Multiome_Pipeline/README.md

```diff
@@ -111,6 +113,16 @@ The Multiome workflow calls two subworkflows, which are described briefly in the
 | gene_metrics_gex | `<input_id>_gex.gene_metrics.csv.gz` | CSV file containing the per-gene metrics. |
 | cell_calls_gex | `<input_id>_gex.emptyDrops` | TSV file containing the EmptyDrops results when the Optimus workflow is run in sc_rna mode. |
 | h5ad_output_file_gex | `<input_id>_gex.h5ad` | h5ad (Anndata) file containing the raw cell-by-gene count matrix, gene metrics, cell metrics, and global attributes. Also contains equivalent ATAC barcode for each gene expression barcode in the `atac_barcodes` column of the `h5ad.obs` property. See the [Optimus Count Matrix Overview](../Optimus_Pipeline/Loom_schema.md) for more details. |
+| cell_barcodes_csv | Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| checkpoint_file | Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentaion](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| h5_array |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| html_report_array |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| log |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| metrics_csv_array |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| output_directory |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
+| summary_pdf |  Optional output. See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). | See [CellBender documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html). |
```

Comment:
```suggestion
| cell_barcodes_csv | `<cell_csv>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| checkpoint_file | `<ckpt_file>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| h5_array | `<h5_array>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| html_report_array | `<report_array>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| log | `<log>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| metrics_csv_array | `<metrics_array>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| output_directory | `<output_dir>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
| summary_pdf | `<pdf>` | Optional output produced when `run_cellbender` is "true"; see CellBender [documentation](https://cellbender.readthedocs.io/en/latest/usage/index.html) and [WDL](https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.1/wdl/cellbender_remove_background.wdl) for more information. |
```

@ekiernan I think it might be useful in this case to name the variable we are pulling the output from in the CellBender WDL so people know what to look for. I know they are similar, but I think this just simplifies things. I also linked the WDL here for ease of looking at how these things are named.